sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -462,3 +462,20 @@ INSERT INTO list_options (list_id,option_id,title) VALUES ('lists','apps','Apps'
 INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity) VALUES ('apps','*OpenEMR','main/main_screen.php',10,1,0);
 INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity) VALUES ('apps','Calendar','main/calendar/index.php',20,0,0);
 #EndIf
+
+#IfNotColumnType list_options list_id varchar(100)
+ALTER TABLE `list_options` CHANGE `list_id` `list_id` VARCHAR(100) NOT NULL DEFAULT '';
+#EndIf
+
+#IfNotColumnType list_options option_id varchar(100)
+ALTER TABLE `list_options` CHANGE `option_id` `option_id` VARCHAR(100) NOT NULL DEFAULT '';
+#EndIf
+
+#IfNotColumnType layout_options list_id varchar(100)
+ALTER TABLE `layout_options` CHANGE `list_id` `list_id` VARCHAR(100) NOT NULL DEFAULT '';
+#EndIf
+
+#IfNotColumnType layout_options list_backup_id varchar(100)
+ALTER TABLE `layout_options` CHANGE `list_backup_id` `list_backup_id` VARCHAR(100) NOT NULL DEFAULT '';
+#EndIf
+
```

Comment:
changing the list_id and list_backup_id seems like overkill here. Isn't just the option_id the only item that needs to be increased?